web/ui/static/js/targets.js

```diff
@@ -1,38 +1,62 @@
-function toggle(obj, state){
-    var icon = $(obj).find("i");
-    if (icon.length === 0 ) {
-        return;
+function toggle(button, state){
```

Comment:
How about `toggleJobTable()` to make the action clearer (since there's different kinds of toggling happening on the page)?